### ISSUE TITLE
[UI] Add lineage sections to Model Detail views

### ISSUE THREAD

**Issue by prodonjs** [kubeflow/metadata#81]
/kind feature
/area front-end

Retrieve and display related Data Set and Evaluation metrics for Models.

[Mocks](https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZUYQ_HerEEyO/files/MCEJu8Y2hyDScfbmC94M0URb19DWmvsbGa4)

**Comment by prodonjs** [comment]
@zhenghuiwang Can you point me to any guidance or documentation on how we might be able to use the new events API methods you added in #53? 

For example, using the data in https://kubeflow-v-n00.endpoints.zhenghui-kubeflow-2.cloud.goog:
`/api/v1alpha1/artifact_types/kubeflow.org/alpha/model/artifacts/40` is the model in question.

`/api/v1alpha1/events/artifacts/40` gives back 
```
{
   "events":[
      {
         "artifact_id":"40",
         "execution_id":"12",
         "type":"OUTPUT",
         "milliseconds_since_epoch":"1562114383006"
      }
   ]
}
```
Then, using the **execution_id** from the returned event, `/api/v1alpha1/events/executions/12` returns 
```
{
   "events":[
      {
         "artifact_id":"39",
         "execution_id":"12",
         "type":"INPUT",
         "milliseconds_since_epoch":"1562114382971"
      },
      {
         "artifact_id":"40",
         "execution_id":"12",
         "type":"OUTPUT",
         "milliseconds_since_epoch":"1562114383006"
      },
      {
         "artifact_id":"41",
         "execution_id":"12",
         "type":"OUTPUT",
         "milliseconds_since_epoch":"1562114383042"
      }
   ]
}
```

Would we then need to go about determining the type of each artifact listed there and retrieving them individually or is there something I am missing?

**Comment by zhenghuiwang** [comment]
You're right. It is a problem: the response doesn't have type info about the artifacts&executions hence you can't retrieve each individual artifact&execution.

Proposed fix: I will add response field `artifacts`: map from id to artifact, and field `executions` similarly.

**Comment by prodonjs** [comment]
blocked by #84

**Comment by zhenghuiwang** [comment]
/reopen

PR#84 is merged and I've updated our dev KF cluster to master. You should see the artifact&execution map in the ListEvent response now.
